### PR TITLE
T-18209 PHD V1 - Use PHD client

### PR DESCRIPTION
**Pull request by commoddity**
MERGE AFTER https://github.com/pokt-foundation/relay-meter/pull/53 E2E TESTS UPDATE

This PR updates Relay Meter to use the PHD client and PHD V1 endpoints.

### CODE REVIEW

**Review by ricantar — DISMISSED** [review]

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,734 @@
+package tests
```
Comment (adshmh) [inline main_test.go]: This looks like a questionable practice. If this needs to be a separate package, then let's have a corresponding directory for this package.

Comment (adshmh) [reply main_test.go]: Also: strongly recommend moving this into its own PR. That will significantly reduce the size of this PR, and allow proper review of both this and the subsequent PR, focusing on the specific aspect they are targeting.

File: cmd/apiserver/main.go [hunk 2]
```diff
@@ -21,144 +20,84 @@ import (
...
-	ENV_TODAYS_METRICS_TTL_SECONDS = "TODAYS_METRICS_TTL_SECONDS"
-	ENV_MAX_ARCHIVE_AGE_DAYS       = "MAX_ARCHIVE_AGE"
-	ENV_SERVER_PORT                = "API_SERVER_PORT"
-
-	LOAD_INTERVAL_DEFAULT_SECONDS      = 30
-	DAILY_METRICS_TTL_DEFAULT_SECONDS  = 120
-	TODAYS_METRICS_TTL_DEFAULT_SECONDS = 60
-	MAX_ARCHIVE_AGE_DEFAULT_DAYS       = 30
-	SERVER_PORT_DEFAULT                = 9898
+	relayMeterAPIKeys = "API_KEYS"
```
Comment (adshmh) [inline cmd/apiserver/main.go]: Thank you for fixing the naming of constants! But this belongs in a different PR.

File: api/meter_test.go [hunk 3]
```diff
@@ -11,10 +11,12 @@ import (
 	"github.com/google/go-cmp/cmp"
 	logger "github.com/sirupsen/logrus"
 
-	"github.com/pokt-foundation/portal-api-go/repository"
+	"github.com/pokt-foundation/portal-db/types"
 	"github.com/pokt-foundation/utils-go/numbers"
 )
 
+var testCtx = context.Background()
```
Comment (adshmh) [inline api/meter_test.go]: Is there a specific reason this is defined once and shared between all test functions?  If not, it is best for each function to define and reuse its own local instance of Context object.

**Review by adshmh — CHANGES_REQUESTED** [review]
Thank you for the PR.
The approach generally looks OK to me. The main blockers are (details in the comments)
- Please move the new test file to a new PR so we can review properly.
- Please either squash the commits into a single (or multiple but single-purpose and independent) commits.
- Please address only a single item in the PR: the changes to use a context along with the new PHD client is a very focused and easy to review change.

File: api/meter_test.go (hunk 3 above)
Comment (commoddity) [reply api/meter_test.go]: fixed

File: cmd/apiserver/main.go (hunk 2 above)
Comment (commoddity) [reply cmd/apiserver/main.go]: updated to use old constant var names in upper snake  case

**Review by adshmh — APPROVED** [review]
